test/infra/anthos-managed/namespaces/secretmanager-csi-build/integration-test-resources.yaml

```diff
@@ -18,6 +18,20 @@ metadata:
 spec:
   displayName: Integration Tests Service Account
 ---
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  name: k8s-csi-test-bindings
+spec:
+  resourceRef:
+    apiVersion: iam.cnrm.cloud.google.com/v1beta1
+    kind: IAMServiceAccount
+    name: k8s-csi-test
+  bindings:
+    - role: roles/iam.workloadIdentityUser
+      members:
+        - serviceAccount:k8s-prow-builds.svc.id.goog[test-pods/secrets-store-csi-driver-gcp]
+---
```

Comment:
Can we add TODO to remove the SA secret too?

Reply:
i made a paper checklist because I also want to remove stuff from test-infra  and internal docs